package.json

```diff
@@ -62,7 +62,7 @@
     "@bbc/psammead-assets": "^2.10.3",
     "@bbc/psammead-brand": "^5.0.11",
     "@bbc/psammead-bulletin": "^1.0.0",
-    "@bbc/psammead-bulleted-list": "0.1.0-alpha.1",
+    "@bbc/psammead-bulleted-list": "0.2.0-alpha.2",
```

Comment:
Alpha components are not allowed to be merged in Simorgh ⚠️

Reply:
Updated